react-nginx/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# 1. For build React app
+FROM node:10 AS builder
+# Set working directory
+WORKDIR /app
+# Copy all files from current directory to working dir in image
+COPY . .
+# install all package and build project
+RUN npm install && npm run build
```

Comment:
Can you take a look to the other examples with Node and Npm, especially to provide best practices on managing layers between dependencies and sources, and adapte your Dockerfile in this way?
For example 
https://github.com/docker/awesome-compose/blob/master/react-java-mysql/frontend/Dockerfile